### PR TITLE
#652 - Test with the stable v0 custom element polyfill and fix tests in it

### LINKED ISSUE #652: Constructors are failing with v0 custom element polyfill
The behaviour between native and polyfilled custom elements v0 behaves differently with constructors. The problem is because native allows you to extend `HTMLElement` and the polyfill does not.

### PR DESCRIPTION
**Pull request by treshugart**
I removed the v1 polyfill as I'd prefer to test in the more common, stable polyfill and create an
issue for testing with both polyfills separately.

Fixes #652.

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = function (config) {
 
   // Shims for testing.
   config.files = [
-    'node_modules/es6-shim/es6-shim.js',
+    'https://npmcdn.com/webcomponents.js/CustomElements.js'
```
Comment (joscha) [inline karma.conf.js]: is this a good idea? That makes the tests dependent on network.

**Comment by joscha** [comment]
Opera keeps on dying on SL...maybe retry later.

File: karma.conf.js (hunk 1 above)
Comment (treshugart) [inline karma.conf.js]: You're right, I'll make it an NPM dependency.

**Comment by treshugart** [comment]
Someone also experienced the root of this problem in https://github.com/webcomponents/webcomponentsjs/issues/563. Should probably see if we can land this in the polyfill instead. Will keep this open until we do so.

**Comment by treshugart** [comment]
screen shot 2016-07-11 at 2 27 03 pm

Looks like Opera is actually passing.

**Comment by treshugart** [comment]
Going to merge this as is and add a todo to follow up about possible fixes in the polyfills.

**Pull request merged**